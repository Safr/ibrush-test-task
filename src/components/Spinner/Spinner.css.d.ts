export const Spinner: string;
export const spinner: string;
export const Skeleton: string;
export const skeleton: string;
export const cpSkeletonAnimate: string;
