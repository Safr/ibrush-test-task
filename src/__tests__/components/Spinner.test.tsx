import React from 'react';
import { shallow } from 'enzyme';
import Spinner from '../../components/Spinner/Spinner';

describe('Spinner Component', () => {
  test('should render correctly', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper).toMatchSnapshot();
  });
});
