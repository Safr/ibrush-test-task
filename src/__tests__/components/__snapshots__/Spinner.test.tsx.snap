// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should render correctly 1`] = `
<div
  className="undefined undefined"
/>
`;
